doc/lsst.multiprofit/getting_started.rst

```diff
@@ -0,0 +1,158 @@
+
+.. _lsst.multiprofit-sources:
+
+Sources
+-------
+
+Sources are constructed from a list of components:
+
+>>> src = g2f.Source([comp])
+>>> print(src)
+
+.. _lsst.multiprofit-models-and-observations:
+
+Models and Observations
+=======================
+
+MultiProFit's :py:class:`lsst.multiprofit.Modeller` requires a :py:class:`lsst.gauss2d.fit.ModelD` object, which in turn needs at least one :py:class:`lsst.gauss2d.fit.ObservationD`.
+
+.. _lsst.multiprofit-observations:
+
+Observations
+------------
+
+An Observation corresponds to a typical astronomical exposure and consists of an image, its errors (stored as inverse sigma for efficiency), a boolean inverse mask (where values of 1 indicate good pixels), and a channel. In this case, we will use the default channel NONE.
+
+>>> import lsst.gauss2d as g2d
+>>> import numpy as np
+>>>
+>>> shape = np.array((15, 17))
+>>> # Center the component
+>>> comp.centroid.y, comp.centroid.x = shape/2.
+>>> image_data = np.zeros(shape)
+>>> sigma_inv_data = np.ones(shape)
+>>> mask_inv_data = np.ones(shape, dtype=bool)
+>>> channel = g2f.Channel.NONE
+>>> # Initializing these in-line with the observation constructor makes a copy
+>>> # for some reason, and image.data is a read-only attr so numpy operators
+>>> # like += can't be used on it directly
+>>> image = g2d.ImageD(image_data)
+>>> sigma_inv = g2d.ImageD(image_data)
+>>> mask_inv = g2d.ImageB(mask_inv_data)
+>>> observation = g2f.ObservationD(image=image, sigma_inv=sigma_inv, mask_inv=mask_inv, channel=channel)
+
+.. _lsst.multiprofit-models:
+
+Models
+------
+
+Models consist of one or more Sources, one or more Observations, the corresponding PSF model for each observation, and any number of priors. For our first PSF model, we will make a unit Gaussian model only.
+
+>>> psf_gaussian = g2f.GaussianComponent()
+>>> psf_gaussian.ellipse.sigma_x = 1
+>>> psf_gaussian.ellipse.sigma_y = 1
+>>> psf_model = g2f.PsfModel([psf_gaussian])
+>>>
+>>> model = g2f.ModelD(data=g2f.DataD([observation]), psfmodels=[psf_model], sources=[src])
+
+Once constructed, models must have an evaluator set up. This design makes repeated model evaluations more efficient. The simplest evaluator makes an image of the model.
+
+>>> model.setup_evaluators(g2f.EvaluatorMode.image)
+>>> model.evaluate()
+>>> output = model.outputs[0]
+
+Now we will use this output image to initialize our observation:
+
+>>> rng = np.random.default_rng(1)
+>>> background = 0.01
+>>> counts = rng.poisson((background + output)*gain)
+>>> image_data += counts/gain - background
+>>> sigma_inv_data += np.sqrt(counts)/gain
+
+The log-likelihood (and prior log likelihood, which will be zero with no priors specified) can now be evaluated:
+
+>>> from lsst.multiprofit.plotting import plot_model_rgb
+>>> import matplotlib.pyplot as plt
+>>>
+>>> model.setup_evaluators(g2f.EvaluatorMode.loglike_image)
+>>> print(model.evaluate())
+>>> plot_model_rgb(model, stretch=1e-3)
```

Comment:
You never imported `plot_model_rgb`.

Reply:
I'm leaving the plot out for now anyway.